app/controllers/admin_controller.rb

```diff
@@ -199,6 +204,9 @@ def edit_user
     # Update email
     user.email = email
 
+    # Update notes
+    user.notes = permitted_params[:notes].strip[0...5000]
```

Comment:
Rather than just cutting things off, this should behave like other models - and have a direct validation: https://github.com/SaraAlert/SaraAlert/blob/master/app/models/patient.rb#L15

Reply:
Fixed in https://github.com/SaraAlert/SaraAlert/pull/805/commits/e79c5dfce756880c55693a245c84a72e7208331c